json/Side_Text.txt

```diff
@@ -28412,270 +28412,270 @@
-		"tr_text": "",
+		"tr_text": "I-is that true, Leader? Woah...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "……チラリズムを解するとは<%br>流石だわ、リーダー。",
-		"tr_text": "",
+		"tr_text": "...So you appreciate the subtle sensuality of a<%br>brief flash of white, do you? How cultured of you.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "そうなんですか？　リーダー。<%br>え、違う？",
-		"tr_text": "",
+		"tr_text": "Is that true, Leader?<%br>She's just jumping to conclusions, isn't she?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "ま、いいわ……今日の任務でも<%br>たっぷりアピールしちゃうんだから。",
-		"tr_text": "",
+		"tr_text": "Well, that's fine... I'll have plenty of<%br>time to put you under my spell today.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "15",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "セクシーでエレガントなアタシの戦い<%br>よーく見ててよね？",
-		"tr_text": "",
+		"tr_text": "I'll show you my sexy, elegant fighting style.<%br>You will keep your eyes on me, won't you?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "ブルージーレクイエムさんは<%br>リーダーが好きなんですか？",
-		"tr_text": "",
+		"tr_text": "Bluesy Requiem, do you like Leader?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "あら、聞かなきゃ分からないほど<%br>適当なアプローチに見えたかしら。",
-		"tr_text": "",
+		"tr_text": "My, my. After all you've seen,<%br>did you really still have to ask?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "あ……ごめんなさい。<%br>そういう意味じゃなくて……",
-		"tr_text": "",
+		"tr_text": "Ah... sorry.<%br>In that case, are you saying...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "……分かってるわよぉ。<%br>からかってるだけに見えるんでしょ？",
-		"tr_text": "",
+		"tr_text": "...I understand where you're coming from.<%br>It must just look like harmless flirting to you.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "まあしょうがないわね。<%br>最初は確かにからかい半分だったしぃ。",
-		"tr_text": "",
+		"tr_text": "I'm no good at expressing how I really feel.<%br>Besides, it really was just a bit of fun at first.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "だけど今は本気よ。素敵な人じゃない。<%br>アタシだけのものにしたくなっちゃったわ。",
-		"tr_text": "",
+		"tr_text": "But now I'm serious, and I don't intend to play<%br>nice. Leader will be mine, and mine alone.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "それでね……メロッメロにして<%br>めちゃくちゃにしちゃうんだからぁ！",
-		"tr_text": "",
+		"tr_text": "Oh, yes... By the time I'm through, I'll have<%br>Leader wrapped around my little finger!",
```

Comment:
`And by the time I'm done, Leader will be nothing but a lifeless husk of burning love!`
(She's implying that after seducing you, you'd be a lifeless drained soul. Ero-manga succubus style)